excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod

```diff
@@ -0,0 +1,270 @@
+
+for (Person p : in.persons){
+ Record r = new Record(["firstname","lastname"], [p.firstname,p.lastname]);
+ out.exportRecordset.add(r);
+}' #txt
+Ps0 f11 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>recordset mapping</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f11 168 202 112 44 -52 -8 #rect
+Ps0 f12 109 224 168 224 #arcP
+Ps0 f13 processCall excel_connector/WriteExcel:call2(Recordset) #txt
+Ps0 f13 requestActionDecl '<Recordset recordset> param;' #txt
+Ps0 f13 requestMappingAction 'param.recordset=in.exportRecordset;
+' #txt
+Ps0 f13 responseMappingAction 'out=in;
+out.excelFile=result.excelFile;
+' #txt
+Ps0 f13 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>excel_connector/WriteExcel</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f13 304 202 160 44 -74 -8 #rect
+Ps0 f14 280 224 304 224 #arcP
+Ps0 f18 guid 17CFF85A10B9B06F #txt
+Ps0 f18 actionTable 'out=in;
+' #txt
+Ps0 f18 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>addPerson</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f18 83 315 26 26 -30 15 #rect
+Ps0 f19 339 315 26 26 0 12 #rect
+Ps0 f21 actionTable 'out=in;
+' #txt
+Ps0 f21 actionCode 'import java.io.FileInputStream;
+import java.io.InputStream;
+import org.primefaces.model.DefaultStreamedContent;
+
+InputStream stream;
+stream = new FileInputStream(in.excelFile.getJavaFile());
+out.streamedExcelFile = new DefaultStreamedContent(stream,"application/vnd.ms-excel","demo_export.xls");' #txt
+Ps0 f21 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>set streamed file</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f21 520 202 112 44 -46 -8 #rect
+Ps0 f22 464 224 520 224 #arcP
+Ps0 f10 632 224 691 224 #arcP
+Ps0 f26 guid 17D03B6D64213F9C #txt
+Ps0 f26 actionTable 'out=in;
+' #txt
+Ps0 f26 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>importExcel</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f26 75 403 26 26 -32 15 #rect
+Ps0 f27 547 403 26 26 0 12 #rect
+Ps0 f29 processCall excel_connector/ReadExcel:call(File) #txt
+Ps0 f29 requestActionDecl '<File excelFile> param;' #txt
+Ps0 f29 requestMappingAction 'param.excelFile=in.importFile;
+' #txt
+Ps0 f29 responseMappingAction 'out=in;
+out.importRecordset=result.recordset;
+' #txt
+Ps0 f29 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>excel_connector/ReadExcel</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f29 144 394 160 44 -75 -8 #rect
+Ps0 f30 304 416 360 416 #arcP
+Ps0 f31 actionTable 'out=in;
+' #txt
+Ps0 f31 actionCode 'import com.axonivy.connector.excel.demo.Person;
+
+for(Record r : in.importRecordset.getRecords()){
+	Person p = new Person();

```

Comment:
next task: can we implement it generically ... without referring 'person' fields?